code/modules/virus2/effect/wordlists.dm

```diff
@@ -0,0 +1,71 @@
+//For lists of words to be used with viruses.
+var/global/list/virus_opposite_word_list
+
+/datum/disease2/proc/initialize_word_lists()
+	virus_opposite_word_list = list("to"		=	"from",				//1
```

Comment:
You should make the first line here start blank and only indent the actual list content by 1 tab.